web/frontend/src/pages/session/Logged.tsx

```diff
@@ -12,14 +12,30 @@ const Logged: FC = () => {
   const authCtx = useContext(AuthContext);
   const fctx = useContext(FlashContext);
 
+  const getCookie = () => {
+    let name = 'redirect' + '=';
+    let decodedCookie = decodeURIComponent(document.cookie);
```

Comment:
There is a simpler way to retrieve a cookie, check the official doc: https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#example_2_get_a_sample_cookie_named_test2